biz.aQute.repository/src/aQute/bnd/deployer/repository/LocalIndexedRepo.java

```diff
@@ -168,7 +236,7 @@ private synchronized void generateIndex(File indexFile, IRepositoryContentProvid
 	}
 
 	@SuppressWarnings("deprecation")
-	private void gatherFiles(Set<File> allFiles) throws Exception {
+	void gatherFiles(Set<File> allFiles) throws Exception {
```

Comment:
Why did this method loose `private`? I did not notice `gatherFiles` used anywhere in this PR.